File: bee1164.py
n_teste = int(input())

for i in range(n_teste):
    soma = 0
    numero = int(input())
    for j in range(1,numero):
        if numero % j == 0:
            soma += j
    if soma == numero:
        print(f"{numero} eh perfeito")
    else:
        print(f"{numero} nao eh perfeito")

File: bee1164.c
#include <stdio.h>
 
int main() {
 
   int n_teste, numero;
   scanf("%d",&n_teste);
   for(int i = 0; i < n_teste; i++){
    scanf("%d",&numero);
    int soma = 0;
    for(int j = 1; j < numero; j++){
        if (numero % j == 0){
            soma += j;

        }
    }
    if(soma == numero){
        printf("%d eh perfeito\n",numero);
    }
    else{
        printf("%d nao eh perfeito\n",numero);
    }
   }
    return 0;
}